src/lambda/snsToSlack.ts

```diff
@@ -0,0 +1,102 @@
+import * as https from 'https';
+import * as util from 'util';
+
+interface PostDataAttachment {
+  color?: string;
+  text?: string;
+  fields?: {
+    title: string;
+    value: string;
+    short: boolean;
+  }[];
+}
+
+interface SlackHookPostData {
+  username: string;
+  text: string;
+  icon_emoji: string;
+  attachments: PostDataAttachment[];
+}
+
+export const snsToSlackHandler = async (event: any, context: any): Promise<void> => {
+  const { SLACK_WEBHOOK } = process.env;
+
+  const message = JSON.parse(event?.Records[0]?.Sns?.Message || {});
+  const region = event?.Records[0]?.EventSubscriptionArn?.split(':')[3];
+  const { AlarmName } = message;
+  const severity = message?.NewStateValue === 'OK' ? 'good' : 'warning';
+  const cwUrl =
+    'https://console.aws.amazon.com/cloudwatch/home?region=' +
+    region +
```

Comment:
Is it possible for `region` const to not resolve correctly?

Reply:
everything is possible :)

Reply:
will fix this by checking if region is there, and filtering out the cwurl if region is for some reason missing.